test/components/util/util.go

```diff
@@ -385,3 +386,13 @@ func IsPlatformSupported(t *testing.T, platforms []Platform) bool {
 
 	return false
 }
+
+// TestNamespacePrefix is testing namespace prefix.
+const TestNamespacePrefix = "test-"
+
+// IsTestingNamespace checks if namespace is testing or not.
+func IsTestingNamespace(name string) bool {
+	matched, _ := regexp.Match(`test-.*`, []byte(name))
```

Comment:
Instead of ignoring the error here, I suggest using `regexp.Must()`,

Also, please use the introduced const for this check, so we don't duplicate the content.

Also, I think adding `^` at the beginning of regex is important, as otherwise things like `calico-test-foo` would also match.